pkg/oci/load_balancer.go

```diff
@@ -70,6 +70,10 @@ const (
 	// ServiceAnnotaionLoadBalancerSecurityListManagementMode is a Service annotation for
 	// specifying the security list managment mode ("All", "Frontend", "None") that configures how security lists are managed by the CCM
 	ServiceAnnotaionLoadBalancerSecurityListManagementMode = "service.beta.kubernetes.io/oci-load-balancer-security-list-management-mode"
+
+	// ServiceAnnotationLoadBalancerEnableHTTPProtocol is a Service annotation for
+	// enabling/disabling the load balancer listener with the http protocol.
```

Comment:
It would be nice if the comment stated what the accepted values are, and also what is the effect of setting this, i.e. why would a user want to do it/why are we supporting it?